AndelaOSP/shufflebox-frontend-react
# Configure Heroku autodeployment

#### What does this PR do?
Configures the application for Heroku autodeployment
#### Description of Task to be completed?
- Create a webpack configuration for production
- Configure production server
#### How should this be manually tested?
- Visit the link https://shufflebox-frontend.herokuapp.com
- You should see the deployed app from this branch
